### DISCUSSION TITLE
Make time granularity selectable

### DISCUSSION THREAD

**Discussion by ribber** [davish/obsidian-full-calendar#421] [Feature Requests]
Currently, each block in the calendar represents 30 minutes. It would be great to have the option to select a finer granularity such as 10 or 15 minutes, similar to Outlook.

**Comment by davish** [comment]
This is the config option that would need to change: https://fullcalendar.io/docs/slotDuration

**Reply by ribber** [reply]
Sorry, I don't know where to configure it. Is it in data.json? I configured it but it didn't work.

**Comment by YouFoundJK** [comment]
This show now be possible in my [fork](https://github.com/YouFoundJK/plugin-full-calendar/releases/latest) by the standard `Ctrl + Scrolling`.
